bemlint/lib/rules/element-inner-parent-block.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const { ClassType } = require('../lint/classType');
+
+module.exports = {
+  id: 'element-inner-parent-block',
+  type: ClassType.Element,
+  check(className, {
+    node,
+    parentBlocks,
+    config: {
+      elementDivider,
+    },
+  }) {
+    const parentBlock = className.split(elementDivider)[0];
+
+    if (!parentBlocks.includes(parentBlock)) {
+      return {
+        id: this.id,
+        node,
+        className,
+        massage: 'Element isn\'t inside the parent block',
+      };
+    }
```

Comment:
would be great to have some convenient way to say there are no errors(like return null)